x-pack/plugins/fleet/server/services/security/uninstall_token_service/index.test.ts

```diff
@@ -9,7 +9,11 @@ import { createHash } from 'crypto';
 
 import type { KibanaRequest } from '@kbn/core-http-server';
 
-import type { SavedObjectsClientContract } from '@kbn/core/server';
+import {
```

Comment:
Could you add some more tests here? It would be great to cover some of the functionalities added below. It will give us more confidence, for instance in the event of the feature flag clean up.

Reply:
Just added some in https://github.com/elastic/kibana/pull/190741/commits/6aa8e8b0528e51a991cf644edfe8d9a0d5d179bd